packages/components/src/banner/banner.tsx

```diff
@@ -36,10 +38,12 @@ const Base = styled(View, {
   alignItems: 'center',
   justifyContent: 'space-between',
   maxHeight: '40px',
+  gap: 10,
 })
 
 const Content = styled(View, {
   flexDirection: 'row',
   gap: 10,
   alignItems: 'center',
+  width: '90%', // truncate does work without this ¯\_(ツ)_/¯
```

Comment:
truncate does or doesn't work without this?

Reply:
Maybe, I am missing something, but this is how it looks without it.

<img width="985" alt="image" src="https://user-images.githubusercontent.com/14926950/230041544-249b7e79-5800-4a62-8ec6-73cfbdb41550.png">

I mean, flexible truncate in the flexbox is always complicated, AFAIK 😅.

Reply:
And with specified widht
<img width="1023" alt="image" src="https://user-images.githubusercontent.com/14926950/230041874-585a7c90-331a-4837-b0c5-3dab4328688e.png">

Reply:
I am actually just referring to comment, it says "truncate does work without this", which doesn't makes sense to me, but it should probably say "truncate doesn't work without this", right?

Reply:
🤦 lol

Reply:
done